package.json

```diff
@@ -37,14 +37,16 @@
         "Visualization"
     ],
     "activationEvents": [
+        "onLanguage:kusto",
```

Comment:
Now supporting language features & execution of queries from kql/csl files (not just notebooks)